core/src/main/java/de/evoila/cf/broker/service/impl/BindingServiceImpl.java

```diff
@@ -212,14 +213,18 @@ public ServiceInstanceBindingResponse syncCreateBinding(String bindingId, Servic
 
 		ServiceInstanceBinding binding;
 		if (haProxyService != null && serviceInstanceBindingRequest.getAppGuid() == null &&
-					(serviceInstanceBindingRequest.getBindResource() != null && serviceInstanceBindingRequest.getBindResource().getAppGuid() == null)) {
+				(serviceInstanceBindingRequest.getBindResource() != null && serviceInstanceBindingRequest.getBindResource().getAppGuid() == null)) {
 			List<ServerAddress> externalServerAddresses = haProxyService.appendAgent(serviceInstance.getHosts(), bindingId, instanceId);
 
 			binding = bindServiceKey(bindingId, serviceInstanceBindingRequest, serviceInstance, plan, externalServerAddresses);
 		} else {
 			binding = bindService(bindingId, serviceInstanceBindingRequest, serviceInstance, plan);
 		}
-
+		if ((catalogService.getServiceDefinition(serviceInstance.getServiceDefinitionId()).isSensitive())) {
```

Comment:
Are we sure at this point that catalogService.getServiceDefinition does not return null?